index.html

```diff
@@ -1,19 +1,71 @@
-</p><h3>Todo</h3>
-<ul id='incompleteTasks'>
-<li><input type='checkbox'><label class="task">Pay Bills</label><input type="text"><button class="edit">Edit</button><button class="delete"><img src="./remove.svg"></button></li>
-<li class="editMode"><input type="checkbox"><label class="task">Go Shopping</label><input type="text" value="Go Shopping" class="task"><button class='edit'>Save</button><button class="delete"><img src="./remove.svg"></button></li>
-</ul><h3>Completed</h3><ul id="completed-tasks"><li><input type="checkbox" checked><label class="task">See the Doctor</label><input type="text" class="task"><button class="edit">Edit</button><button class="delete"><img src="./remove.svg"></button>
-</li>
-</ul>
-</div>
-<script type="text/javascript" SRC="app.js"></script>
+  <header class="header">
+    <img src="./eisenhower-matrix.jpg" class="header__img" alt="Logo Eisenhower Matrix">
+    <a class="header__link" href="https://goal-life.com/page/method/matrix-eisenhower">
+      Want more details?
+    </a>
+  </header>
+  <main class="todo-list">
+    <section class="new-task">
+      <label class="new-task__label" for="new-task">Add Item</label>
+      <p class="new-task__wrapper">
+        <input id="new-task" class="new-task__input" type="text">
+        <button class="new-task__btn btn">Add</button>
+      </p>
+    </section>
+    <section>
+      <h3 class="title">Todo</h3>
+      <ul class="task-list" id="incomplete-tasks">
+        <li class="task-list__item">
+          <input type="checkbox" class="task-list__checkbox">
+          <label class="task-list__label">Pay Bills</label>
+          <input type="text" class="task-list__input">
+          <button class="task-list__edit btn">Edit</button>
+          <button class="task-list__delete btn">
+            <img src="./remove.svg" alt="Delete icon">
+          </button>
+        </li>
+        <li class="task-list__item edit-mode">
+          <input type="checkbox" class="task-list__checkbox">
+          <label class="task-list__label">Go Shopping</label>
+          <input type="text" value="Go Shopping" class="task-list__input">
+          <button class="task-list__edit btn">Save</button>
+          <button class="task-list__delete btn">
+            <img src="./remove.svg" alt="Delete icon">
+          </button>
+        </li>
+      </ul>
+    </section>
+    <section class="completed-tasks">
+      <h3 class="title">Completed</h3>
+      <ul class="task-list" id="completed-tasks">
+        <li class="task-list__item">
+          <input type="checkbox" class="task-list__checkbox" checked>
+          <label class="task-list__label">See the Doctor</label>
+          <input type="text" class="task-list__input">
+          <button class="task-list__edit btn">Edit</button>
+          <button class="task-list__delete btn">
+            <img src="./remove.svg" alt="Delete Icon">
```

Comment:
можно что-то такое добавить
```suggestion
            <img class="btn__icon" src="./remove.svg" alt="Delete Icon">
```